requirements_doc.txt

```
@@ -1,7 +1,7 @@
 # requirements for building docs
 sphinx
 https://github.com/numpy/numpydoc/archive/main.zip
-pydata-sphinx-theme==0.6.1
+pydata-sphinx-theme>=0.6.3
```

I would keep this pinned.  The theme is still under pretty active development, and its test suite is still being refined.
```suggestion
pydata-sphinx-theme==0.6.3
```

---

I assumed that was the original rationale, but I'm hoping that by doing this we can catch their bugs earlier, similar to how we build CircleCI with `sphinx-gallery:master`. I figure if it starts breaking things in a way we can't easily fix then we can easily add a fixed pin later

---

fair enough.  I guess staying on latest point release is not as dangerous as being on their latest main, anyway.